### DISCUSSION TITLE
Add Google Chrome more prominent as requirement

### DISCUSSION THREAD

**Discussion by onmax** [harlan-zw/unlighthouse#71] [Ideas]
I have been trying to run the tool but I am had trouble trying to install it. And it took me about 20 minutes to realize that maybe I need to install Google Chrome, since with Brave nor Edge is possible.

I am using WSL 2 but I also try it in my Windows Machine. None of those worked until I installed Google Chrome..

I have try it with
```bash
npx unlighthouse --site google.com
pnpm dlx unlighthouse --site google.com
npx unlighthouse-puppeteer --site google.com
```

I would suggest adding a clearer banner or some text saying that Google Chrome is required (I thought with chromium-based browser was enough). I know is my fault but I am a noob, and other people like me could have the same issue.

I love the tool btw :heart:

**Comment by harlan-zw** [comment]
It's a good idea, I did have plans to improve the browser resolution so even if you don't have chrome installed it won't fail. Still a WIP though.

**Comment by harlan-zw** [comment]
In version 0.7.0 a fallback Chromium will be automatically downloaded which should solve this. See https://github.com/harlan-zw/unlighthouse/releases/tag/v0.7.0
